itest/performance/mint_batch_test.go

```diff
@@ -0,0 +1,193 @@
+package perf_test
+
+import (
+	"context"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	_ "embed"
+
+	"github.com/btcsuite/btcd/rpcclient"
+	"github.com/lightninglabs/taproot-assets/fn"
+	"github.com/lightninglabs/taproot-assets/itest"
+	"github.com/lightninglabs/taproot-assets/taprpc"
+	"github.com/lightninglabs/taproot-assets/taprpc/mintrpc"
+	unirpc "github.com/lightninglabs/taproot-assets/taprpc/universerpc"
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed testdata/8k-metadata.hex
+var imageMetadataHex []byte
```

Comment:
Very nice, wasn't aware that you could directly load into bytes, thought you always needed to use `embed.Fs`.
That reminds me, would be nice to have a `make` goal to compile this test binary so we don't have to do that manually.